spec/latest/core/index.html

```diff
@@ -1039,6 +1039,44 @@ <h3>Templated Links</h3>
       }
       -->
     </pre>
+
+    <p>Iri expansion should be performed with respect to the specification
+      behind the iri template type (which is RFC 6570 by default), and the
+      product of this process SHOULD be an Iri. In may happen that this
+      product is a relative Uri. In such a case client by default SHOULD
+      stick to RFC 3986. It is due to fact that most RDF serialization
+      allowing other that absolute Iris will behave that way and the result
+      of Iri template expansion should be compliant. Still, it may be
```

Comment:
> It is due to fact that most RDF serialization allowing other that absolute Iris will behave that way and the result of Iri template expansion should be compliant

what did you mean here?

Reply:
Also, I would specifically mention that sections `5.1.3` and `5.1.4` must be followed to **establish base URI** as the RFC puts it. 

Those are the relevant ones as we discussed that neither `5.1.1` and `5.1.2` can apply in our context.